apps/cloud/app/(private)/contacts/[id]/import/page.tsx

```
@@ -0,0 +1,363 @@
+                                }
+                              >
+                                {option.label}
+                                {isDisabled && ' (Already mapped)'}
+                              </SelectItem>
+                            );
+                          })}
+                        </SelectContent>
+                      </Select>
+                    </TableCell>
+                    <TableCell>
+                      {csvData.rows[0] && csvData.rows[0][index] ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)